README.md

```diff
@@ -35,6 +35,11 @@ C:\> iex ((New-Object System.Net.WebClient).DownloadString('https://raw.githubus
 
 Look at the [cli/](cli/) folder for more information.
```

Comment:
Can we move this to the end of this section? That is, move the new Homebrew section to line `35`, please.

Reply:
Will do!